pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
     "fastapi",
     "pydantic",
     "pydantic-settings",
-    "python-dateutil==2.9.0",
+    "python-dateutil==2.9.0.post0",
```

Comment:
@ericbuckley does this seem like a weird version change to you?...

Reply:
seems legit when looking into it

Reply:
Yeah, just seems like how they handle bugfixes.  https://github.com/dateutil/dateutil/releases